schnorr_fun/src/frost/chilldkg.rs

```diff
@@ -1019,19 +1063,21 @@ pub mod certpedpop {
         /// can use the share you must call [`finalize`] with a completed certificate.
         ///
         /// [`finalize`]: Self::finalize
-        pub fn receive_share<H, NG>(
+        pub fn receive_share<H, NG, S>(
```

Comment:
We should rename this to receive_secret_share and call this SecretShareReceiver.